app/registries/setting_registry.rb

```diff
@@ -0,0 +1,30 @@
+class SettingRegistry
```

Comment:
I'd keep it simple for now, as it is used during the initialization process, I've used simple singleton
Later I'd like to hava lazy initialization and it could be thread safe, every instance would load the settings from the Setting classes itself.